src/views/Projects/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { Component, Fragment } from 'react';
+import { withStyles } from 'material-ui/styles';
+import Grid from 'material-ui/Grid';
+import Button from 'material-ui/Button';
+import Typography from 'material-ui/Typography';
+import projects from '../../data/loader';
+import ProjectCard from '../../components/ProjectCard';
+
+@withStyles(
+  theme => ({
+    grid: {
```

Comment:
Remove this class. You can just add a spacing prop to a Grid container https://material-ui.com/layout/grid/#grid.

Reply:
Ok!